assets/js/modules/analytics/datastore/tags.js

```diff
@@ -292,11 +280,10 @@ export const selectors = {
 	 *
 	 * @param {Object} state      Data store's state.
 	 * @param {string} propertyID The Analytics Property ID to check permissions for.
-	 * @param {string} accountID  Optional. The Analytics Account ID the property belongs to, if known.
 	 * @return {?boolean} True if the user has access, false if not; `undefined` if not loaded.
 	 */
-	hasTagPermission: createRegistrySelector( ( select ) => ( state, propertyID, accountID = '' ) => {
-		const { permission } = select( STORE_NAME ).getTagPermission( state, propertyID, accountID ) || {};
+	hasTagPermission: createRegistrySelector( ( select ) => ( state, propertyID ) => {
+		const { permission } = select( STORE_NAME ).getTagPermission( state, propertyID ) || {};
```

Comment:
I don't think we should pass `state` to `getTagPermission` here since this is called from the selector bound to the store, it should already have `state` provided to it. We should only need to pass that through if we were calling the unbound selector directly (i.e. `selectors.getTagPermission(...)`)

Reply:
That is correct! Anytime you're using `registry.select( $storeName ).$someSelector()`, `state` is automatically added to the selector as the first argument. This should fail tests if the response is expected to be anything other than `undefined` (and might even fail then).

```suggestion
	hasTagPermission: createRegistrySelector( ( select ) => ( state, propertyID ) => {
		const { permission } = select( STORE_NAME ).getTagPermission( propertyID ) || {};
```

I'm also not a fan of defaulting to the empty object either. I understand it technically does the same thing, but I think it makes the code/control flow a lot less explicit. Compare the above to:

```suggestion
	hasTagPermission: createRegistrySelector( ( select ) => ( state, propertyID ) => {
		const tagPermission = select( STORE_NAME ).getTagPermission( propertyID );
		
		return tagPermission ? tagPermission.permission : undefined;
```

I think the latter is more explicit.